lib/DocumentsFieldArray/FileUploaderField.js

```diff
@@ -50,12 +60,14 @@ class FileUploaderField extends React.Component {
 
     this.props.onUploadFile(acceptedFiles[0])
       .then(response => {
-        if (response.error) {
-          throw new Error(`${response.message} (${response.error})`);
+        if (response.ok) {
+          response.json().then(resp => {
```

Comment:
Nit-picking (since this'll need a respin anyway 😆) but this could be a tad cleaner if you called the variable `resp` instead of `file`.